.github/workflows/run-tests-split.yml

```diff
@@ -20,6 +20,9 @@ on:
       working_directory:
         type: string
         default: .
+      pytest_rootdir:
```

Comment:
By speeding up the testsuite, I was able to revert the `run-tests-split` change. We can decide its fate later on.